web/modules/custom/swap_blocks/src/Plugin/paragraphs/Behavior/ImageAndTextBehavior.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\swap_blocks\Plugin\paragraphs\Behavior;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\paragraphs\Entity\Paragraph;
+use Drupal\paragraphs\Entity\ParagraphsType;
+use Drupal\paragraphs\ParagraphInterface;
+use Drupal\paragraphs\ParagraphsBehaviorBase;
+
+/**
+ * Some short description.
+ *
+ * @ParagraphsBehavior(
+ *   id = "dlog_paragraphs_image_and_text",
+ *   label = @Translation("Image and text settings"),
+ *   description= @Translation("Settings for image and text paragraph type."),
+ *   weight = 0,
+ * )
+ */
+class ImageAndTextBehavior extends ParagraphsBehaviorBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function isApplicable(ParagraphsType $paragraphs_type) {
+    return $paragraphs_type->id() == 'image_and_text';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function view(array &$build, Paragraph $paragraph, EntityViewDisplayInterface $display, $view_mode) {
+    $image_position = $paragraph->getBehaviorSetting($this->getPluginId(), 'image_position', 'left');
+    $build['#attributes']['class'][] = Html::getClass('paragraph-' . 'image-position-' . $image_position);
+
+    $hide_for_mobile = $paragraph->getBehaviorSetting($this->getPluginId(), 'hide_for_mobile', 0);
+
+    if ($hide_for_mobile == 0) {
+      $build['#attributes']['class'][] = Html::getClass('paragraph-' . 'mobile-' . 'visible');
+    }
+    else {
+      $build['#attributes']['class'][] = Html::getClass('paragraph-' . 'mobile-' . 'hide');
+    }
+  }
```

Comment:
$is_hidden = $hide_for_mobile ? hide : visible;
$build['#attributes']['class'][] = Html::getClass('paragraph-' . 'mobile-' . $is_hidden. ....

change class name should be hidden and visible